README.md

```diff
@@ -51,45 +51,36 @@ Install it
 
     bundle install
 
-### Rails 4.x.x
+Add to your `app/assets/javascripts/application.js` file
 
-Run the generator
-
-    rails generate autocomplete:install
-
-And include autocomplete-rails.js on your layouts
-
-    javascript_include_tag "autocomplete-rails.js"
```

Comment:
Technically, this gem is for Rails 3 AND 4, as it is the currently maintained repo. I think it is best to condense any configuration into 1 section (not by Rails Version), and just add a note: `When updating the gem, it is best to regenerate the javascript file, to ensure all fixes are made` or something like that